beacon-chain/rpc/eth/events/events.go

```diff
@@ -711,7 +711,7 @@ func (s *Server) fillEventData(ctx context.Context, ev payloadattribute.EventDat
 			return ev, errors.Wrap(err, "could not get head state")
 		}
 		// double check that we need to process_slots, just in case we got here via a hot state cache miss.
-		if slots.ToEpoch(st.Slot()) == pse {
+		if slots.ToEpoch(st.Slot()) < pse {
```

Comment:
For the record, this was never the intention, it was a typo in https://github.com/OffchainLabs/prysm/pull/15213 -- thank you for fixing!